pages/c/_cluster/explorer/members/index.vue

```diff
@@ -62,6 +63,7 @@ export default {
     <Masthead
       :schema="schema"
       :resource="resource"
```

Comment:
Not related to changes in this PR, but for `Cluster Members` the side nav and favourite entries are nicely named `Cluster Members` whilst the title of the list/edit page is `ClusterRoleTemplateBindings`. Could probably get around it with a custom `typeLabel.management.cattle.io.clusterroletemplatebinding`?

Reply:
Yeah, I noticed this regression in my current feature so I fixed it there. Since you noticed it here I'll go ahead and add it.